autolamella/structures.py

```diff
@@ -981,6 +981,46 @@ def save(self, path: Path) -> None:
         """Save the protocol to disk."""
         with open(path, "w") as f:
             yaml.safe_dump(self.to_dict(), f, indent=4)
+
+    def save_base_protocol(self, path: Path, current_milling_dict: dict = None) -> None:
+        """Saves less verbose version of full experiment protocol, 
+        similar to base protocol for methods"""
+        with open(path, "w") as f:
+            yaml.safe_dump(self.to_dict_base_protocol(current_milling_dict), f, indent=4)
+
+    def to_dict_base_protocol(self,current_milling_dict=None):
+
+        """Keep only base necessary items to not overwhelm the protocol yaml file"""
+
+        if current_milling_dict is None:
+            current_milling_dict = {k: get_protocol_from_stages(v) for k, v in self.milling.items()}
+
+        milling_dict = self._reduced_milling_options_dict(current_milling_dict) 
```

Comment:
I don't agree with always removing these options from the dict, as others use the alignment/imaging settings. Instead of specifying method.name =="Waffle" (L1013), can you just add a flag here to remove the options for your use case. If it stays the way it is, it's gonna cause some silent issues later on